README.md

```diff
@@ -73,14 +73,23 @@ options := make(map[string]interface{})
 options["revenueValue"] = 12
 isSuccessful = vwoInstance.Track(campaignKey, userID, goalIdentifier, options)
 
-// With both Custom Variables and Revenue Value
+// With Custom Variables, Revenue Value, GoalTypeToTrack and ShouldTrackreturningUser
 options := make(map[string]interface{})
 options["customVariables"] = map[string]interface{}{"a": "x"}
 options["revenueValue"] = 12
+//  Set specific goalType to Track
+//  Available GoalTypes - constants.GoalTypeRevenue, constants.GoalTypeCustom, constants.GoalTypeAll (Default)
+options["goalTypeToTrack"] = "ALL"
```

Comment:
"ALL", "CUSTOM", "REVENUE" are values of enum...in other SDKs we have exposed our internal enum as well so that user won't mistake typing it. Have we exposed this enum? If not, please; else let's use this here

Reply:
Yes they are exposed and can be used as stated in the above line 81.
Used !